README.md

```diff
@@ -29,7 +31,7 @@ Instructions about how to build and install the Solidity compiler can be found i
 A "Hello World" program in Solidity is of even less use than in other languages, but still:
 
 ```
-pragma solidity ^0.4.16;
```

Comment:
Someone requested to use the pragma range as wide as possible to show when certain features were introduced. I'm fine to relax this for this code snippet - but I'm not sure we will be able to keep it up to date. Perhaps make this `^0.5.0`? We could also extend the compilation tests in `isolate_tests.py` to cover the readme - this will check if it can be compiled with the current version.

Reply:
Good compromise, updated in https://github.com/fulldecent/solidity/commit/0842ac66880ce78b332753005e6407262464e0da

Created separate issue to track that update at https://github.com/ethereum/solidity/issues/5767